### DISCUSSION TITLE
我想一个程序配置两个SS给两个人用,但是配置启动总错误,大神帮看看是怎么回事

### DISCUSSION THREAD

**Discussion by linsan970809** [v2fly/v2ray-core#2497] [General]
我是想启动两个SS给不同的人用,端口也不同,但是这个配置总提示错误,但是我这个是按照网上的教程给的配置写的,不知道是哪里出了问题
截图左边是启动时的错误提示,右边是V2RAY的配置文件
![image](https://user-images.githubusercontent.com/61045370/236516341-912d7f13-549b-4765-87c6-b17d04f3fc69.png)
感谢大神

**Comment by mydogshitgold** [comment]
The error message is quite obvious:
Your json has syntax errors and the comma at the end of line 25 should be removed. Read [json.org docs](https://www.json.org/json-en.html) if you don't know how to write proper JSON.

Notepad++ is a good software overall, but lacks certain features that make it easier to edit JSON files. Consider using vscode, which supports syntax checking, auto formating and autocomplete when provided with [schema](https://github.com/EHfive/v2ray-jsonschema), for this task.

---

`inboundDetour` and `outboundDetour` are deprecated. It may be a good idea to migrate them.

---

Since there is no user level policy anywhere in this config, the `inbound[].settings.level` can be omitted.

**Reply by linsan970809** [reply]
非常感谢您的回复,还帮我指出其它的错误.谢谢

**Comment by qianbinbin** [comment]
25 行的 `,` 去掉
建议发文字，前后用 ``` 包裹，图片看起来很费劲
